server/go/api/appencryption.pb.go

```diff
@@ -0,0 +1,776 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
do we need to commit this self generated code?

Reply:
Definitely. Committing the code makes for consistent, dependable builds and also drastically lowers the barrier of entry for those looking to try out the server.